engine/dlib/src/dlib/connection_pool.cpp

```diff
@@ -149,6 +149,8 @@ namespace dmConnectionPool
                 break;
             case STATE_INUSE:
                 stats->m_InUse++;
+                if (c->m_Socket != dmSocket::INVALID_SOCKET_HANDLE)
+                    stats->m_InUseAndValid++;
```

Comment:
This stat function isn't used for anything other that for a test, but I needed to synchronize the test somehow.